js/alerts.js

```diff
@@ -66,9 +66,9 @@ $(function mitlib_alerts(){
 				// Alert HTML template
 				alert_template = 	'<div class="posts--preview--alerts transition-vertical transition-vertical--hide">' +
 														'<div class="post alert--critical flex-container">' +
-															'<svg class="icon-exclamation-circle width="2048" height="2048" viewBox="0 0 2048 2048" xmlns="http://www.w3.org/2000/svg"><path d="M1024 256q209 0 385.5 103t279.5 279.5 103 385.5-103 385.5-279.5 279.5-385.5 103-385.5-103-279.5-279.5-103-385.5 103-385.5 279.5-279.5 385.5-103zm128 1247v-190q0-14-9-23.5t-22-9.5h-192q-13 0-23 10t-10 23v190q0 13 10 23t23 10h192q13 0 22-9.5t9-23.5zm-2-344l18-621q0-12-10-18-10-8-24-8h-220q-14 0-24 8-10 6-10 18l17 621q0 10 10 17.5t24 7.5h185q14 0 23.5-7.5t10.5-17.5z"/></svg>' +
+																'<i class="icon-exclamation-sign" aria-hidden="true"></i>' +
```

Comment:
Changes to this file are for switching from svg to font-icon for the sitewide alert to fix a safari bug.